javascript-forms/form-values.js

```diff
@@ -0,0 +1,14 @@
+const $contactForm = document.querySelector('#contact-form');
+const formValues = {};
+
+function completedForm(event) {
+  formValues.name = $contactForm['user-name'].value;
+  formValues.email = $contactForm['user-email'].value;
+  formValues.message = $contactForm['user-message'].value;
+
+  console.log(formValues);
+  event.preventDefault();
+  document.querySelector('#contact-form').reset();
```

Comment:
You already have a variable for this, why not use that? No need to do another `querySelector`